### PR TITLE
fix BigSur VPN bug

### PR DESCRIPTION
**Pull request by asakapab0i**
## Description of the change

This commit contains the fixes of BigSur OS VPN connection bug

## Type of change
- [x] Bug fix
- [ ] New feature

### Testing

- [ ] Testing information has been added - test cases checklist and steps followed for testing.
- [x] Testing screenshot(s) attached for more information.

The fix has been confirmed, this requires additional testing

## Security

- [ ] This PR has security related considerations.

### CODE REVIEW

File: templates/ipsec.conf [hunk 1]
```diff
@@ -21,7 +21,8 @@ conn vpnpsk
   type=transport
   ike=3des-sha1,aes-sha1
   ikev2=no
-  phase2alg=3des-sha1,aes-sha1
+  phase2alg=3des-sha1,aes-sha1,aes256-sha2
```
Comment (aren) [inline templates/ipsec.conf:24]: Was this change required? @ttheune said he wasn't sure it was necessary.

File: templates/ipsec.conf [hunk 2]
```diff
@@ -21,7 +21,8 @@ conn vpnpsk
...
+  sha2-truncbug=no
```
Comment (aren) [inline templates/ipsec.conf:25]: Once this change is made, does this mean the VPN is no longer compatible with OSX Catalina?

File: templates/ipsec.conf (hunk 1 above)
Comment (asakapab0i) [reply templates/ipsec.conf:24]: This is the error being displayed.
https://support.apple.com/en-us/HT211840

File: templates/ipsec.conf (hunk 2 above)
Comment (asakapab0i) [reply templates/ipsec.conf:25]: I'd have to contact the customer again and get more information about the old device and if he can connect again using it.

Comment (ttheune) [reply templates/ipsec.conf:25]: This has now been tested on Catalina and Big Sur.

File: templates/ipsec.conf (hunk 1 above)
Comment (ttheune) [reply templates/ipsec.conf:24]: Will be going with an expanded phase2alg list with the xauth build soon, so this is okay either way.

File: requirements.txt [hunk 3]
```diff
@@ -5,3 +5,4 @@ pyasn1==0.4.8
 python-hosts==1.0.0
 pyOpenSSL==19.1.0
 requests==2.23.0
+greenlet==0.4.15
```
Comment (ttheune) [inline requirements.txt]: remove this and leave it simply in the foxpass-radius-agent

Comment (aren) [reply requirements.txt]: should probably be `gevent` in `requirements.txt` for the `foxpass-radius-agent`, not `greenlet`.

**Review by ttheune — APPROVED** [review]
looks good
